flaml/nlp/utils.py

```diff
@@ -272,7 +274,7 @@ def tokenize_text_multiplechoice(X, custom_hpo_args):
     return output, None
 
 
-def tokenize_swag(this_row, this_tokenizer, custom_hpo_args):
+def tokenize_swag(this_row, this_tokenizer, custom_hpo_args=None):
```

Comment:
rename all "this_tokenizer"
```suggestion
def tokenize_swag(this_row, tokenizer, custom_hpo_args=None):
```